content/engine/release-notes/27.1.md

```diff
@@ -4,13 +4,6 @@ description: Learn about the new features, bug fixes, and breaking changes for D
 keywords: docker, docker engine, ce, whats new, release notes
 toc_min: 1
 toc_max: 2
-tags:
```

Comment:
Considering if we should've used a "v27" file so that we could put all minor versions in there 🤔 maybe that ship sailed, or perhaps we could still combine them; @dvdksn WDYT?

Reply:
yeah should be fine to consolidate all v27 to the same page. We can alias the paths with minor versions.

Reply:
I updated this PR with this consolidation